### ISSUE TITLE
crash [1.19.2]

### ISSUE THREAD

**Issue by Xplodin** [TheWinABagel/Zenith#37]
https://gist.github.com/Xplodin/3c0c7c9f32c14bee8a4ee8eb5f106149

using AOF6 with a bunch of added mods

**Comment by CMD-God** [comment]
I've been having the same issue, but with a completely different modpack.

**Comment by Shibva** [comment]
Same here; By the looks of it the error is a fatal mixin injection that failed

````[20:28:02] [main/ERROR]: Minecraft has crashed!
net.fabricmc.loader.impl.FormattedException: java.lang.BootstrapMethodError: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1685 failed
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:63) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.14.21.jar:?]
Caused by: java.lang.BootstrapMethodError: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1685 failed
	at net.minecraft.class_1299.<clinit>(class_1299.java:187) ~[client-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[client-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:84) ~[client-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:20) ~[client-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:24) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_44104(class_5458.java:98) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:105) ~[client-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:104) ~[client-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:326) ~[client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:50) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:161) ~[fabric-loader-0.14.21-1.19.2.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[fabric-loader-0.14.21-1.19.2.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468) ~[fabric-loader-0.14.21.jar:?]
	... 2 more
Caused by: java.lang.RuntimeException: Mixin transformation of net.minecraft.class_1685 failed
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:427) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:112) ~[fabric-loader-0.14.21.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_1299.<clinit>(class_1299.java:187) ~[client-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[client-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:84) ~[client-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:20) ~[client-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:24) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_44104(class_5458.java:98) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:105) ~[client-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:104) ~[client-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:326) ~[client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:50) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:161) ~[fabric-loader-0.14.21-1.19.2.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[fabric-loader-0.14.21-1.19.2.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468) ~[fabric-loader-0.14.21.jar:?]
	... 2 more
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:112) ~[fabric-loader-0.14.21.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_1299.<clinit>(class_1299.java:187) ~[client-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[client-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:84) ~[client-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:20) ~[client-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:24) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_44104(class_5458.java:98) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:105) ~[client-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:104) ~[client-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:326) ~[client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:50) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:161) ~[fabric-loader-0.14.21-1.19.2.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[fabric-loader-0.14.21-1.19.2.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468) ~[fabric-loader-0.14.21.jar:?]
	... 2 more
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [zenith.mixins.json:ThrownTridentMixin from mod zenith] from phase [DEFAULT] in config [zenith.mixins.json] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:638) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:589) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:112) ~[fabric-loader-0.14.21.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_1299.<clinit>(class_1299.java:187) ~[client-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[client-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:84) ~[client-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:20) ~[client-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:24) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_44104(class_5458.java:98) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:105) ~[client-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:104) ~[client-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:326) ~[client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:50) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:161) ~[fabric-loader-0.14.21-1.19.2.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[fabric-loader-0.14.21-1.19.2.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468) ~[fabric-loader-0.14.21.jar:?]
	... 2 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: @Inject annotation on init could not find any targets matching '<init>(Lnet/minecraft/world/level/Level;Lnet/minecraft/world/entity/LivingEntity;Lnet/minecraft/world/item/ItemStack;)V' in net.minecraft.class_1685. Using refmap zenith-refmap.json [PREINJECT Applicator Phase -> zenith.mixins.json:ThrownTridentMixin from mod zenith -> Prepare Injections ->  -> handler$ofi023$zenith$init(Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;)V -> Parse]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.validateTargets(InjectionInfo.java:656) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.findTargets(InjectionInfo.java:587) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.readAnnotation(InjectionInfo.java:330) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:316) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.<init>(InjectionInfo.java:308) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.injection.struct.CallbackInjectionInfo.<init>(CallbackInjectionInfo.java:46) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at jdk.internal.reflect.GeneratedConstructorAccessor22.newInstance(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:?]
	at java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499) ~[?:?]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:480) ~[?:?]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo$InjectorEntry.create(InjectionInfo.java:149) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.parse(InjectionInfo.java:708) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.prepareInjections(MixinTargetContext.java:1329) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.prepareInjections(MixinApplicatorStandard.java:1053) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:327) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202) ~[sponge-mixin-0.12.5+mixin.0.8.5.jar:0.12.5+mixin.0.8.5]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:422) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:323) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:218) ~[fabric-loader-0.14.21.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:112) ~[fabric-loader-0.14.21.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.minecraft.class_1299.<clinit>(class_1299.java:187) ~[client-intermediary.jar:?]
	at net.minecraft.class_3103.<clinit>(class_3103.java:28) ~[client-intermediary.jar:?]
	at net.minecraft.class_3031.<clinit>(class_3031.java:84) ~[client-intermediary.jar:?]
	at net.minecraft.class_6800.<clinit>(class_6800.java:20) ~[client-intermediary.jar:?]
	at net.minecraft.class_6803.method_39702(class_6803.java:24) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_44104(class_5458.java:98) ~[client-intermediary.jar:?]
	at net.minecraft.class_5458.method_30566(class_5458.java:105) ~[client-intermediary.jar:?]
	at java.util.LinkedHashMap.forEach(LinkedHashMap.java:721) ~[?:?]
	at net.minecraft.class_5458.<clinit>(class_5458.java:104) ~[client-intermediary.jar:?]
	at net.minecraft.class_2378.<clinit>(class_2378.java:326) ~[client-intermediary.jar:?]
	at net.minecraft.class_2966.method_12851(class_2966.java:50) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:161) ~[fabric-loader-0.14.21-1.19.2.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) ~[fabric-loader-0.14.21-1.19.2.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468) ~[fabric-loader-0.14.21.jar:?]
	... 2 more````

**Comment by palapapa** [comment]
Same error

**Comment by TheWinABagel** [comment]
Will be fixed in 0.6.5, damn mixins breaking only out of dev environment
